calyx-py/test/source-loc/invoke-with-builder.expect

```diff
@@ -45,7 +45,7 @@ component foo<"pos"={4}>(a: 32) -> (out: 32) {
 
 sourceinfo #{
 FILES 
-  0: /home/ayaka/projects/calyx/calyx-py/test/source-loc/invoke-with-builder.py
+  0: /home/calyx/calyx-py/test/source-loc/invoke-with-builder.py
```

Comment:
ugh, this is a bit awkward. Can we add an option to only include relative paths? We need to ensure that tests can pass on different machines.

Reply:
I made it configurable via the environment variable and updated tests, so hopefully that will do the trick? [commit is here](https://github.com/calyxir/calyx/commit/9d241f9269a2246fcb68c5471db07f4c7d36761d)